src/hooks/motionWidgets/stakingWidget/useTotalStakeRadios.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import { defineMessages } from 'react-intl';
+
+import {
+  StakeSide,
+  useStakingWidgetContext,
+} from '~common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget';
+import { useAppContext } from '~hooks';
+import { CustomRadioAppearance } from '~shared/Fields/Radio';
+import Icon from '~shared/Icon';
+import Numeral from '~shared/Numeral';
+
+const displayName = 'hooks.motionWidgets.stakingWidget.useTotalStakeRadios';
+
+const MSG = defineMessages({
+  YAYName: {
+    id: `${displayName}.YAYName`,
+    defaultMessage: 'Motion {fullyStakedEmoji}',
+  },
+  NAYName: {
+    id: `${displayName}.NAYName`,
+    defaultMessage: 'Objection {fullyStakedEmoji}',
+  },
+  stakeProgress: {
+    id: `${displayName}.stakeProgress`,
+    defaultMessage: '{totalPercentage}% of {requiredStake}',
+  },
+  fullyStaked: {
+    id: `${displayName}.fullyStaked`,
+    defaultMessage: 'Fully staked',
+  },
+});
+
+const useTotalStakeRadios = () => {
+  const {
+    yayPercentage,
+    nayPercentage,
+    nativeTokenSymbol,
+    nativeTokenDecimals,
+    requiredStake,
+  } = useStakingWidgetContext();
+  const { getValues } = useFormContext();
+  const { stakeSide } = getValues();
+  const { user } = useAppContext();
+  const isYAYSideFullyStaked = yayPercentage === '100';
+  const isNAYSideFullyStaked = nayPercentage === '100';
+
+  const radioConfig = [
```

Comment:
Motion related "widgets" will stay more or less the same after the UI refresh as well, so it very good that you extracted this in a hook, as this will definitely be used even after we change the interface